docs/docs/about/references/keywords/EMISSIONS.md

```diff
@@ -45,6 +45,7 @@ EMISSIONS:
 
 ## For venting emitters (type: DIRECT_EMISSION)
 Note that the emission name is case-insensitive.
+The keywords [NAME](/about/references/keywords/NAME.md) and [RATE](/about/references/keywords/RATE.md) are required.
```

Comment:
Could/should mention from which version they are required? Since a user may have an older version...?